src/test/java/cloud/fogbow/ras/core/OrderControllerTest.java

```diff
@@ -91,57 +80,47 @@ public void testDeleteClosedOrderThrowsInstanceNotFoundException()
         this.ordersController.deleteOrder(computeOrder);
     }
 
-    // test case: Checks if getInstancesStatus() returns exactly the same list of instances that
+    // test case: Checks if the getInstancesStatusmethod returns exactly the same
+    // list of instances that
     // were added on the lists.
     @Test
     public void testGetAllInstancesStatus() throws InstanceNotFoundException {
         // set up
-        SystemUser systemUser = new SystemUser("fake-id", "fake-user", "fake-token-provider"
-        );
+        SystemUser systemUser = createSystemUser();
 
-        ComputeOrder computeOrder = new ComputeOrder();
-        computeOrder.setSystemUser(systemUser);
-        computeOrder.setRequester(this.localMember);
-        computeOrder.setProvider(this.localMember);
-        computeOrder.setOrderStateInTestMode(OrderState.FULFILLED);
+        ComputeOrder computeOrderFulfilled = createLocalComputeOrder();
+        computeOrderFulfilled.setSystemUser(systemUser);
+        computeOrderFulfilled.setOrderStateInTestMode(OrderState.FULFILLED);
 
-        ComputeOrder computeOrder2 = new ComputeOrder();
-        computeOrder2.setSystemUser(systemUser);
-        computeOrder2.setRequester(this.localMember);
-        computeOrder2.setProvider(this.localMember);
-        computeOrder2.setOrderStateInTestMode(OrderState.FAILED_AFTER_SUCCESSFUL_REQUEST);
+        ComputeOrder computeOrderFailed = createLocalComputeOrder();
+        computeOrderFailed.setSystemUser(systemUser);
+        computeOrderFailed.setOrderStateInTestMode(OrderState.FAILED_AFTER_SUCCESSFUL_REQUEST);
```

Comment:
I removed all uses of the setOrderStateInTestMode method used in the test classes in this branch. The mockReadOrdersFromDataBase method in the BaseUnitTests class allows the change to the setOrderState method without problems.

Unfortunately, I cannot remove the setOrderStateInTestMode method because it is used by many test classes.

Perhaps flagging this method with @Deprecated will allow better identification to cease its use and gradually remove it from the other test classes.

Reply:
I'll leave a comment about this method so that it will be removed when it is no longer used.